### ISSUE TITLE
Some interfaces are hidden from `deno doc`

### ISSUE THREAD

**Issue by solson** [denoland/deno#7472]
I noticed that `deno doc --builtin WebSocket` only shows this:

```ts
var WebSocket: { prototype: WebSocket; CLOSED: number; CLOSING: number; CONNECTING: number; OPEN: number; }
```

While if you browse through `deno types` output, you can find much more detail:

<details>
<summary>(Folded for length)</summary>

```ts
/** Provides the API for creating and managing a WebSocket connection to a server, as well as for sending and receiving data on the connection. */
interface WebSocket extends EventTarget {
  /**
   * Returns a string that indicates how binary data from the WebSocket object is exposed to scripts:
   *
   * Can be set, to change how binary data is returned. The default is "blob".
   */
  binaryType: BinaryType;
  /**
   * Returns the number of bytes of application data (UTF-8 text and binary data) that have been queued using send() but not yet been transmitted to the network.
   *
   * If the WebSocket connection is closed, this attribute's value will only increase with each call to the send() method. (The number does not reset to zero once the connection closes.)
   */
  readonly bufferedAmount: number;
  /**
   * Returns the extensions selected by the server, if any.
   */
  readonly extensions: string;
  onclose: ((this: WebSocket, ev: CloseEvent) => any) | null;
  onerror: ((this: WebSocket, ev: Event | ErrorEvent) => any) | null;
  onmessage: ((this: WebSocket, ev: MessageEvent) => any) | null;
  onopen: ((this: WebSocket, ev: Event) => any) | null;
  /**
   * Returns the subprotocol selected by the server, if any. It can be used in conjunction with the array form of the constructor's second argument to perform subprotocol negotiation.
   */
  readonly protocol: string;
  /**
   * Returns the state of the WebSocket object's connection. It can have the values described below.
   */
  readonly readyState: number;
  /**
   * Returns the URL that was used to establish the WebSocket connection.
   */
  readonly url: string;
  /**
   * Closes the WebSocket connection, optionally using code as the the WebSocket connection close code and reason as the the WebSocket connection close reason.
   */
  close(code?: number, reason?: string): void;
  /**
   * Transmits data using the WebSocket connection. data can be a string, a Blob, an ArrayBuffer, or an ArrayBufferView.
   */
  send(data: string | ArrayBufferLike | Blob | ArrayBufferView): void;
  readonly CLOSED: number;
  readonly CLOSING: number;
  readonly CONNECTING: number;
  readonly OPEN: number;
  addEventListener<K extends keyof WebSocketEventMap>(type: K, listener: (this: WebSocket, ev: WebSocketEventMap[K]) => any, options?: boolean | AddEventListenerOptions): void;
  addEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: boolean | AddEventListenerOptions): void;
  removeEventListener<K extends keyof WebSocketEventMap>(type: K, listener: (this: WebSocket, ev: WebSocketEventMap[K]) => any, options?: boolean | EventListenerOptions): void;
  removeEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: boolean | EventListenerOptions): void;
}

declare var WebSocket: {
  prototype: WebSocket;
  new(url: string, protocols?: string | string[]): WebSocket;
  readonly CLOSED: number;
  readonly CLOSING: number;
  readonly CONNECTING: number;
  readonly OPEN: number;
};
```

</details>

The presence of `declare` makes the difference (`export` also seems to work). We can reproduce this locally:

```console
$ cat example1.ts
interface Foo { x: number };
var Foo: { y: string };

$ deno doc example1.ts

$ cat example2.ts
interface Foo { x: number };
declare var Foo: { y: string };

$ deno doc example2.ts
Defined in file:///home/scott/Projects/dotfiles/scripts/docbug/example2.ts:2:8

var Foo: { y: string; }


$ cat example3.ts
declare interface Foo { x: number };
var Foo: { y: string };

$ deno doc example3.ts
Defined in file:///home/scott/Projects/dotfiles/scripts/docbug/example3.ts:1:0

interface Foo

  x: number


$ cat example4.ts
declare interface Foo { x: number };
declare var Foo: { y: string };

$ deno doc example4.ts
Defined in file:///home/scott/Projects/dotfiles/scripts/docbug/example4.ts:2:8

var Foo: { y: string; }

Defined in file:///home/scott/Projects/dotfiles/scripts/docbug/example4.ts:1:0

interface Foo

  x: number
```

The doc website has [the same problem](https://doc.deno.land/builtin/v1.4.0#WebSocket), as far as I can tell.

**Comment by solson** [comment]
I see a mixture of `declare interface` and `export interface` occurring in `deno types`, but I haven't learned what this means in TypeScript yet:

```ts
declare interface ImportMeta {
```

```ts
  export interface TestDefinition {
```

**Comment by bartlomieju** [comment]
@solson can we close this issue?

**Comment by solson** [comment]
Yes, fixed by #7514.
